runner/startup.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+
+# SC1091: source /opt/bash-utils/logger.sh won't exist when shellcheck is ran
+# shellcheck disable=SC1091
+
```

Comment:
FYI TIL that you can tell shellcheck the path to look from the scripts.
`shellcheck --source-path runner` made this mark unnecessary.